src/coreclr/src/tools/crossgen2/ILCompiler.ReadyToRun/Compiler/DependencyAnalysis/ReadyToRun/DebugInfoTableNode.cs

```diff
@@ -70,6 +71,42 @@ public override void AppendMangledName(NameMangler nameMangler, Utf8StringBuilde
             sb.Append("__ReadyToRunDebugInfoTable");
         }
 
+        private class DebugInfoSignature
```

Comment:
I wonder whether there might be a chance to simplify some of this using the existing ByteArrayComparer.

Reply:
Thanks Tomas; that does exactly what I need. I'll bin the `DebugInfoSignature`.